editor/project_settings_editor.cpp

```diff
@@ -133,6 +135,9 @@ void ProjectSettingsEditor::_notification(int p_what) {
 				translation_file_open->add_filter("*." + E->get());
 			}
 
+			translation_pot_file_open->add_filter("*.tscn"); // THIS ONE NEEDS TO BE CHANGED
```

Comment:
Indeed, matches my previous comment about using `ResourceLoader::get_recognized_extensions_for_type` like done just below.

Same for `.gd` maybe, in case we add more valid extensions for the `GDScript` resource type in the future.